File: Exercises/T3/src/java/Cargos/Assistente.java
package Exercises.T3.src.java.Cargos;
import Exercises.T3.src.java.GestaoABC.Funcionaria;

public class Assistente extends Funcionaria {
  public Assistente(String nome, String CPF, double salarioBase) {
    this.nome = nome;
    this.CPF = CPF;
    this.salarioBase = salarioBase;
  };

  public double calculaSalario() {
    return this.salarioBase;
  };

  @Override
  public String toString() {
    String str = String.format("Cargo: Assistente%n");
    str += this.superToString();
    return str;
  }
}

File: Exercises/T3/src/python/Cargos/Assistente.py
from GestaoABC.Funcionaria import Funcionaria

class Assistente(Funcionaria):
    def __init__(self, nome: str, CPF: str, salarioBase: float) -> None:
        super().__init__(nome, CPF, salarioBase)

    def calculaSalario(self) -> float:
        return self.salarioBase
    
    def __str__(self) -> str:
        return "Cargo: Assistente\n" + super().__str__()
